src/components/tutorial/TutorialNav.astro

```diff
@@ -54,7 +54,8 @@ const isCurrentUnit = (unit: typeof units[number]) =>
 									</div>
 									{currentUrl.endsWith(lesson.slug) && (
 										<ul class="on-this-page">
-											{(await lesson.getHeadings())
```

Comment:
Unrelated, but this made an hint in `astro check`, since the await is unneeded (`getHeadings()` is apparently sync)

Reply:
The types say so but I seem to recall I actually needed the `await` when building that, probably worth double checking.

Reply:
Seems to work: 
<img width="304" alt="image" src="https://user-images.githubusercontent.com/3019731/212676697-9d20780a-ce84-4ca8-841f-1bd885873290.png">

Reply:
Nice — thanks for checking!